### ISSUE TITLE
Why is this not installable with pip?

### ISSUE THREAD

**Issue by mihaelacr** [jlowin/pyautodiff#8]
This seems like a great library and as a person who knows theano I was thinking of doing something like this myself to avoid manually differentiating functions.

People would be a lot keener to use it if it were installable with pip. I think you have done some steps to be able to upload the project to PyPI, why also not do the rest? Is there a problem with the licensing once you upload it?

Thanks for writing this btw :)

**Comment by drewid** [comment]
I agree.  As a relative newbie to python I am not sure how to do so without pip

**Comment by cgc** [comment]
I was able to install this for Python 2 with
```shell
pip install -e git+git@github.com:LowinData/pyautodiff.git@python2-ast#egg=autodiff
```
while I haven't tested this, you can probably install it for Python 3 with
```shell
pip install -e git+git@github.com:LowinData/pyautodiff.git@python3#egg=autodiff
```

**Comment by RiccardoRiglietti** [comment]
I found the command to install in Python 3 with pip:

`pip install -e "git+https://github.com/jlowin/pyautodiff/#egg=autodiff"`
